usecase/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation 'commons-cli:commons-cli:1.4'
     implementation 'org.apache.logging.log4j:log4j-api:2.12.1'
     implementation 'org.apache.logging.log4j:log4j-core:2.12.1'
+    implementation 'commons-validator:commons-validator:1.6'
```

Comment:
We should standardize on regex implementations. Apache commons is fine, but if we use that, we should use it everywhere.

Reply:
Good point. I guess we shall open an issue of tracking down current regex usage?

Reply:
See https://github.com/MobilityData/gtfs-validator/issues/305